indexers/vector/FaissIndexer/__init__.py

```diff
@@ -35,6 +36,7 @@ def __init__(self,
         :param train_filepath: the training data file path, e.g ``faiss.tgz`` or `faiss.npy`. The data file is expected
             to be either `.npy` file from `numpy.save()` or a `.tgz` file from `NumpyIndexer`.
         :param distance: 'l2' or 'inner_product' accepted. Determines which distances to optimize by FAISS
+        :param normalize: whether or not to normalize the vectors (e.g. for the cosine similarity)
```

Comment:
I think this new param is not needed, the normalization is needed if distance is l2 according to current parameters.

Reply:
Do you really think so @JoanFM? My team actually wanted to try out l2 and the inner_product without normalization. Normalization removes the magnitude and this can be counterproductive in many applications.

Reply:
Maybe setting the normalization to True as a default would be better?

Reply:
my bad, looking back at the documentation it says not to confuse inner product with cosine similarity which is what I was doing here ;).

I would keep normalize to False as default. So good as it is